core/src/main/java/de/sesu8642/feudaltactics/menu/statistics/ui/StatisticsEventHandler.java

```diff
@@ -0,0 +1,64 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+package de.sesu8642.feudaltactics.menu.statistics.ui;
+
+import javax.inject.Inject;
+
+import com.google.common.eventbus.Subscribe;
+
+import de.sesu8642.feudaltactics.events.GameExitedEvent;
+import de.sesu8642.feudaltactics.lib.gamestate.GameState;
+import de.sesu8642.feudaltactics.lib.gamestate.Player;
+import de.sesu8642.feudaltactics.lib.gamestate.Player.Type;
+import de.sesu8642.feudaltactics.lib.ingame.botai.Intelligence;
+import de.sesu8642.feudaltactics.menu.statistics.StatisticsDao;
+
+/**
+ * Handles events for the statistics screen.
+ */
+public class StatisticsEventHandler {
```

Comment:
Calling it StatisticsScreenEventHandler would be more consistent with the other handlers and the class description. But I think you chose this name because you also call the DAO from here.

In other cases (see PreferencesScreenEventHandler), I called the screen from the event handler and the screen then called the DAO. This way, the EventHandlers can stay very small. I think I prefer it that way.
But to be fair, I overused Events a bit and want to to simplify. So consider this comment a suggestion.